lib/components/user/places/place-editor.tsx

```diff
@@ -139,7 +139,12 @@ class PlaceEditor extends Component<
 
         <FlexContainer>
           {/* For fixed places, just show the icon for place type instead of all inputs and selectors */}
-          {isFixed && <FixedPlaceIcon type={place.icon} />}
+          {/* {isFixed && <FixedPlaceIcon type={place.icon} />} */}
+          {isFixed && (
+            <FixedPlaceIconWrapper size="1.5x" spaceRight>
```

Comment:
Why `1.5x`?

Reply:
IIRC I just experimented in the inspector until it looked right...